src/main/java/frc/robot/Robot.java

```diff
@@ -124,6 +125,10 @@ public void autonomousInit() {
     if (m_autoCommand != null) {
       m_autoCommand.schedule();
     }
+
+    // Makes it trust vision more if it goes over the charge station
+    // TODO: Change this to whatever commands do that
+    m_robotContainer.setChargeStationAuto(m_autoCommand instanceof DefaultDriveCommand);
```

Comment:
What if it is another command that isn't DefaultDriveComand but also isn't the charge station?

Reply:
I would just add this as part of the auto routine

Reply:
This will be changed to something else, not DefaultDriveCommand. If it doesn't go over the charge station, don't add it here.

Reply:
I don't have the classes for the actual autos.